components/pkg-export-kubernetes/src/cli.rs

```diff
@@ -206,6 +208,16 @@ fn valid_natural_number(val: String) -> result::Result<(), String> {
     }
 }
 
+fn valid_bind(val: String) -> result::Result<(), String> {
+    match ServiceBind::from_str(&val) {
+        Ok(_) => Ok(()),
+        Err(_) => Err(format!(
+            "'{}' is not valid. Service binds have the format name:service.group",
+            &val
+        )),
+    }
+}
```

Comment:
I'd argue for leaving this here for now.  Looking at the k8s exporter code it could probably do with a cleanup generally.  I'll add a comment to say we could move it to core.